src/main/groovy/com/getkeepsafe/dexcount/sdkresolver/SdkResolver.groovy

```diff
@@ -99,26 +115,32 @@ class SdkResolver {
         if (userAndroid.exists()) {
             log.debug "Found existing SDK at '$userAndroid.absolutePath'. Writing to $FN_LOCAL_PROPERTIES."
 
-            writeLocalProperties userAndroid.absolutePath
+            writeLocalProperties(userAndroid.absolutePath, ndkDir)
             return userAndroid
         }
 
         return null
     }
 
-    def writeLocalProperties(String path) {
+    def writeLocalProperties(String sdkPath, String ndkPath) {
         if (isWindows) {
-            // Escape Windows file separators when writing as a path.
-            path = path.replace "\\", "\\\\"
+            // Escape Windows file separators when writing as a sdkPath.
+            sdkPath = sdkPath.replace "\\", "\\\\"
+            if (ndkPath!=null)
```

Comment:
As a point of style, we always use braces around conditionals or loops, so I'd prefer that this read:
```
if (ndkPath != null) {
  ndkPath = ndkPath.replace "\\", "\\\\"
}
```

Likewise for the other two ifs in this method.